docs/manual/advanced/survey_validators.md

```diff
@@ -0,0 +1,93 @@
+# Survey validators
+
+Survey validators are Python modules that can be added as plugin. It allows users to implement their own validation
+logic on a day1 or day2 operation.
```

Comment:
against all field of the survey instead of only one single field with with the "field_validator"

Reply:
Do we want to keep field validator ? Maybe we can stop to maintain field validators and tell users to move on survey validators. They are more user friendly and you can use the same code for both UI and API. 

If we decide to stop field validators in X months, we should not include a citation here. Need to discuss